modules/e2ee/E2EEContext.js

```diff
@@ -341,6 +341,13 @@ export default class E2EEcontext {
                     controller.enqueue(encodedFrame);
                 }
             });
+        } else if (keyIndex >= this._cryptoKeyRing.length
+                && this._cryptoKeyRing[this._currentKeyIndex % this._cryptoKeyRing.length]) {
+            // If we are encrypting but don't have a key for the remote drop the frame.
```

Comment:
decrypting?

Reply:
no, this checks whether *we* have a key set. We don't know what the remote does

Reply:
🤦